src/warpgrad/warpgrad/updaters.py

```diff
@@ -22,8 +22,46 @@
                     state_dict_to_par_list)
 
 
-class DualUpdater:
+class SimpleUpdater:
+    """
+    """
+
+    def __init__(self, criterion, init_objective=0,
+                 epochs=1, bsz=1, norm=True, approx=False):
+        """Initialize an dummy updater.
 
+        Arguments:
+            criterion (function): task loss criterion.
+            init_objective (int): type of objective for initialization
+                (optional).
+            epochs (int): number of times to iterate over buffer (default=1).
+            bsz (int): task parameter batch size between updates (default=1).
+            norm (bool): use the norm in the Leap objective (d1)
+                (default=True).
+            approx (bool): use approximate (Hessian-free) meta-objective.
+        """
+        self.init_objective = init_objective
+        self.criterion = criterion
+        self.epochs = epochs
+        self.approx = approx
+        self.norm = norm
+        self.bsz = bsz
+
+    def backward(self, model, step_fn, **opt_kwargs):
+        """It does nothing for now.
+
+        Arguments:
+            model (Warp): warped model to backprop through.
+            step_fn (function): step function for the meta gradient.
+            **opt_kwargs (kwargs): optional arguments to inner optimizer.
+        """
+
+        # init_objective = INIT_OBJECTIVES[self.init_objective]
+        # init_objective(model.named_init_parameters(suffix=None),
+        #                params, self.norm, self.bsz, step_fn)
+        pass
```

Comment:
I have commented out initialization objective for now. Should we also use leap based initialization for online learning?